.travis.yml

```diff
@@ -10,13 +10,15 @@ git:
   submodules: false
 
 before_install:
-  - sudo apt update && sudo apt install -y python-pip python-dev protobuf-compiler
+  - sudo apt update && sudo apt install -y python-pip python-dev protobuf-compiler python3-pip
   - sed -i 's,git\@github.com:\([^/]*\)/\(.*\).git,https://github.com/\1/\2,' .gitmodules
   - git submodule update --init --recursive
   - sudo pip install -r source/tyr/requirements_dev.txt
+  - pip3 install black
```

Comment:
from what I remember, pip3 uses python 3.5 when Black requires 3.6 :/ 
You need to do something like: `python3.6 -m pip install black`